src/block-data-providers/esplora.provider.ts

```diff
@@ -110,6 +111,7 @@ export class EsploraProvider
                     BitcoinNetwork.MAINNET
                         ? TAPROOT_ACTIVATION_HEIGHT - 1
                         : 0,
+                lastProcessedTxIndex: -1,
```

Comment:
now that i think about it, the default value should be `0`, not `-1`
This is because we want to skip the coinbase transaction